dlpy/network.py

```diff
@@ -1107,6 +1130,28 @@ def load_weights_attr(self, path):
         if not flag:
             self._retrieve_('table.dropcaslib', caslib=cas_lib_name)
 
+    def share_weights(self, layers):
```

Comment:
please do this rather than as to do, we are also working on documentation. at least some explanations so that we can go over it